### PR TITLE
data:lakshmishreea122003

### PR DESCRIPTION
**Pull request by lakshmishreea122003**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4498"></a>

### CODE REVIEW

**Review by sidharthviz — DISMISSED** [review]
Looks Good.

**Review by AbbeyIT — DISMISSED** [review]
LGTM! Thank you for your contribution.

File: data/lakshmishreea122003.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Lakshmi Shree A",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "ML and DevOps enthusiast| Java| Python| Web development| Open Source 🚀",
+  "avatar":"https://github.com/lakshmishreea122003.png",
```
Comment (kumarsonsoff3) [inline data/lakshmishreea122003.json:6]: In our recent update, we no longer need the `avatar` key. So please remove this from your json file. 
```suggestion
```

**Review by kumarsonsoff3 — DISMISSED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**